### PR TITLE
Kogito 5339 update docs consoles

### PR DESCRIPTION
**Pull request by AjayJagan**
Updating examples in accordance with SSO integration in the consoles.

This PR can be merged after https://github.com/kiegroup/kogito-apps/pull/914 has been merged.

It contains the following changs:
 - update Readme in accordance with the new SSO integration.
 - update docker-compose.yml to integrate keycloak server and managment and task console props
 - update the bpmn diagram to use proper `actors`

### CODE REVIEW

**Comment by pefernan** [comment]
Let's have this merged first: https://github.com/kiegroup/kogito-examples/pull/767

File: process-usertasks-springboot-with-console/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -75,19 +92,24 @@ services:
...
     environment:
-      KOGITO_DATAINDEX_HTTP_URL: http://${DOCKER_GATEWAY_HOST}:8180
-      KOGITO_MANAGEMENT_CONSOLE_PROPS: -Dkogito.svg.folder.path=/home/kogito/data/svg
+      KOGITO_DATAINDEX_HTTP_URL: http://${DOCKER_GATEWAY_HOST}:8180/graphql
+      KOGITO_MANAGEMENT_CONSOLE_PROPS: -Dkogito.consoles.keycloak.config.url=http://${DOCKER_GATEWAY_HOST}:8480/auth -Dkogito.consoles.keycloak.config.health-check-url=http://${DOCKER_GATEWAY_HOST}:8480/auth/realms/kogito/.well-known/openid-configuration -Dkogito.svg.folder.path=/home/kogito/data/svg
 
   task-console:
     container_name: task-console
-    image: quay.io/kiegroup/kogito-task-console:${KOGITO_VERSION}
+    image: quay.io/kiegroup/kogito-task-console:${KOGITO_VERSION}/graphql
```
Comment (bxf12315) [inline process-usertasks-springboot-with-console/docker-compose/docker-compose.yml]: Are you sure it's right format? 
"ERROR: no such image: quay.io/kiegroup/kogito-task-console:latest/graphql: invalid reference format"

Comment (AjayJagan) [reply process-usertasks-springboot-with-console/docker-compose/docker-compose.yml]: okay that is my bad , I should remove the graphql :D

Comment (AjayJagan) [reply process-usertasks-springboot-with-console/docker-compose/docker-compose.yml]: removed the typo now!

**Comment by AjayJagan** [comment]
will reopen a new PR with proper branch and name - #783

**Pull request closed without merging**